src/main/java/screens/GraphScene.java

```diff
@@ -105,18 +105,20 @@ private void drawGraphUtil(int center, int radius) {
         LinkedList<DrawNode> drawNodes = nodeGraph.getDrawNodes();
         LinkedList<DummyNode> dummyNodes = nodeGraph.getDummyNodes();
         for (DrawNode dNode : drawNodes) {
+            int count = 0;
```

Comment:
I suggest refactoring the name of count to something more descriptive. That way we know what is being counted.